rpcs3/Emu/RSX/Overlays/overlay_edit_text.cpp

```diff
@@ -5,6 +5,32 @@ namespace rsx
 {
 	namespace overlays
 	{
+		using u64 = size_t;
```

Comment:
This was only necessary in the sample project. You can remove this :)

Reply:
Haha, I am not a cpp dev, so I am quite using what's given. Will remove in a sec.